### PR TITLE
either_or_both: Fix inverted Left/Right in docstrings

### PR DESCRIPTION
**Pull request by cgwalters**
I stared at this for a bit, confused before deciding the docs
must be wrong.

### CODE REVIEW

File: src/either_or_both.rs [hunk 1]
```diff
@@ -140,7 +140,7 @@ impl<A, B> EitherOrBoth<A, B> {
         }
     }
 
-    /// Apply the function `f` on the value `b` in `Right(b)` or `Both(a, _)` variants if it is
+    /// Apply the function `f` on the value `b` in `Left(b)` or `Both(a, _)` variants if it is
```
Comment (phimuemue) [inline src/either_or_both.rs:143]: Shouldn't this be ".. or `Both(b, _)`? And maybe it should use the placeholder `a`?

File: src/either_or_both.rs [hunk 2]
```diff
@@ -153,7 +153,7 @@ impl<A, B> EitherOrBoth<A, B> {
     }
 
     /// Apply the function `f` on the value `a`
-    /// in `Left(a)` or `Both(a, _)` variants if it is present.
+    /// in `Right(a)` or `Both(a, _)` variants if it is present.
```
Comment (phimuemue) [inline src/either_or_both.rs:156]: Shouldn't this use placeholder `b` and be `Both(_, b)`?

**Review by phimuemue — COMMENTED** [review]
Thank you for spotting this. Maybe we should try to bring all the doc in line, using `a` for `Left`, `b` for `Right`?

File: src/either_or_both.rs (hunk 1 above)
Comment (cgwalters) [reply src/either_or_both.rs]: Yeah I think we're `Both` `Right`, that'd `b` `a` better fix.  Updated!

**Comment by phimuemue** [comment]
Thank you.

bors r+

**Pull request merged**